src/status_im/ui/screens/browser/styles.cljs

```diff
@@ -0,0 +1,65 @@
+(ns status-im.ui.screens.browser.styles
+  (:require-macros [status-im.utils.styles :refer [defstyle defnstyle]])
+  (:require [status-im.ui.components.styles :as component.styles]))
+
+(def browser {:flex 1})
+
+(defstyle dapp-name
+  {:flex            1
+   :justify-content :center
+   :margin-left     12
+   :android         {:padding-bottom 6}})
+
+(def dapp-name-text
+  {:color    component.styles/color-gray6
+   :fontSize 16})
+
+(defstyle dapp-text
+  {:color   component.styles/text4-color
```

Comment:
Can we move it to colors?

Reply:
idk how it works? do we have rules or doc?

Reply:
No we build it on the go. If it's a new color and it looks legitimate, then had it with generic name.
If it looks similar to another one then discuss with design team to remove it.